src/main/java/com/shopify/model/ShopifyFulfillmentOrder.java

```diff
@@ -413,4 +413,15 @@ public void setUpdatedAt(DateTime updatedAt) {
 		this.updatedAt = updatedAt;
 	}
 
+	public boolean hasSupportedAction(final SupportedActions action) {
```

Comment:
You can simplify this whole function down to one line:

> return this.getSupportedActions().anyMatch(supportedAction -> supportedAction.equalsIgnoreCase(action.toString()));

It's why i love functional programming

Reply:
I'm still somewhat confused with Java's arrow functions, I stored on my snippets here
you'll see it in my next commits